extensions/amp-story/1.0/amp-story-templates.css

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** position="landscape-half-left" template. */
+[orientation=landscape] [position="landscape-half-left"] {
+  width: 50% !important;
```

Comment:
Not 100% sure, but maybe it's worth either:

1. Leaving a comment here (and below in `landscape-half-right`) about the dependencies you're assuming (`position: absolute; top: 0; bottom: 0`)
2. Alternatively, specifying all the styles you need, even though they're duplicated